lib/util/template/index.ts

```diff
@@ -201,13 +206,17 @@ export function compile(
   input: CompileInput,
   filterFields = true
 ): string {
-  const data = { ...GlobalConfig.get(), ...input };
+  const env = getChildEnv({}) ?? {};
```

Comment:
```suggestion
  const env = getChildEnv({});
```
should never return null or undefined. fix test mock instead

Reply:
I cannot recall why I added it in the first place.

I removed it as suggested, ran the tests and all passed.